### DISCUSSION TITLE
Device config, support for bitsets?

### DISCUSSION THREAD

**Discussion by LordMike** [zwave-js/zwave-js#1393] [Q&A] [answered]
Hi,

Does the format for these device configs, support bitsets? The readme has a todo reg. the schema, and I couldn't immediately find another device with something that looked like bitsets.

For [this configuration parameter](https://github.com/zwave-js/node-zwave-js/blob/7353977fe9457c319bc3a3a46b330316381fbda2/packages/config/config/devices/0x0234/zdb5100.json#L20-L30), the values are actually bits that can be set.

Per our other discussions, it seems like this should be "partial values", each a boolean.. Ie. 0x01 for button 1, 0x02 for button 2, 0x04 for button 3 etc..

Mike.

**Comment by AlCalzone** [comment, marked as answer]
I'm not sure what TODO you mean. 

The way you describe the config param, it seems like it is a perfect fit for partial parameters.

**Reply by LordMike** [reply]
https://github.com/zwave-js/node-zwave-js/blob/8c62cad12e9b0575d7c4428876f4900090468985/packages/config/config/README.md

This todo: 
> TODO: Describe options in the config files

:)

**Reply by AlCalzone** [reply]
Uh, I should probably just remove that. The documentation is complete in that regard.

**Reply by LordMike** [reply]
Found the docs on the site. The readme could probably point to this page:

https://zwave-js.github.io/node-zwave-js/#/development/config-files

:)

**Reply by AlCalzone** [reply]
Hmm the Readme does link to the docs and the sidebar does too 🤷🏻‍♂️

**Reply by LordMike** [reply]
I meant here: https://github.com/zwave-js/node-zwave-js/tree/master/packages/config/config

I'll make a suggestion.. :)

**Comment by robertsLando** [comment]
Yep bitset values in zwave-js are partial parameters
